CMakeLists.txt

```diff
@@ -378,19 +382,6 @@ GR_REGISTER_COMPONENT("testing-support" ENABLE_TESTING
     CPPUNIT_FOUND
 )
 
-# check if CppUnit version is 1.14.0 or greater; requires c++11 ...
-
-if(CPPUNIT_FOUND AND NOT "${CPPUNIT_VERSION}" VERSION_LESS "1.14.0")
-    message(WARNING "\nWarning: CppUnit version is ${CPPUNIT_VERSION} which requires C++11 for building. Trying to set CMake internally to use C++11 ...")
-    SET(CMAKE_CXX_STANDARD 11)
-endif()
-
-# if cmake version is < 3.1, explicitly set C++ standard to use.
-
-if(${CMAKE_VERSION} VERSION_LESS "3.1")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++${CMAKE_CXX_STANDARD}")
```

Comment:
Capitalize for consistency

Reply:
Err, that's a removed `set` (:

Reply:
It's hard to distinguish red and green sometimes. There are a few more places where my comment applies, though.

Reply:
Only one `set(` was in my PR. I pushed a seperate commit fixing every other `set(` in the file for sake of consistency. "Current" cmake coding style for a copule of years now is having everything lowercase though. Emacs users even have a "unscreamify" macro :)